source/iterators.tex

```diff
@@ -604,12 +604,15 @@
 \range{i}{s} denotes a valid range.
 
 \pnum
-A counted range \range{i}{n} is empty if \tcode{n == 0}; otherwise, \range{i}{n}
-refers to the \tcode{n} elements in the data structure starting with the element
-pointed to by \tcode{i} and up to but not including the element, if any, pointed to by
-the result of \tcode{n} applications of \tcode{++i}. A counted range
-\range{i}{n} is valid if and only if \tcode{n == 0}; or \tcode{n} is positive,
-\tcode{i} is dereferenceable, and \range{++i}{-{-}n} is valid.
+A \defnx{counted range}{range!counted} \range{i}{n} is empty if \tcode{n == 0};
```

Comment:
Please use `\defnadj{counted}{range}`, which also adds a cross-reference into the index from "counted range" to "range, counted".